Formula/octave.rb

```diff
@@ -65,7 +65,15 @@ def install
     inreplace "src/mkoctfile.in.cc", /%OCTAVE_CONF_OCT(AVE)?_LINK_(DEPS|OPTS)%/, '""'
 
     args = []
-    args << "--without-qt" if build.without? "qt"
+    if build.with? "qt"
+      # Stuff for Qt 5.12 compatibility
+      # https://savannah.gnu.org/bugs/?55187
+      ENV["QCOLLECTIONGENERATOR"]="qhelpgenerator"
+      ENV.append "CPPFLAGS", "-I#{Formula["qt"].opt_include}"
+      ENV.append "LDFLAGS", "-F#{Formula["qt"].opt_lib}"
```

Comment:
I don't understand the need for `CPPFLAGS` and `LDFLAGS`, given that Homebrew should add these automatically

Reply:
You're right. Those were left over from when I was building it outside Homebrew. Removed.